docs/modules/fs-extra.md

```diff
@@ -0,0 +1,60 @@
+# fs-extra
+
+`fs-extra` adds some file system methods that previously were not included in the native `fs` module and added `Promise` support to the `fs` methods.
+Many of the fs-extra modules have since been implemented natively by Node.
+
+[fs-extra](https://github.com/jprichardson/node-fs-extra)
+
+## Alternatives
+
+### `node:fs`
+
+[Node.js File System Module](https://nodejs.org/docs/latest/api/fs.html)
+
+To use the promise-based APIs:
+
+```js
+import * as fs from 'node:fs/promises';
+```
+
+To use the callback and sync APIs:
+
+```js
+import * as fs from 'node:fs/promises';
+```
+
+### Equivalent Methods
+
+- `ensureFile` -> `fs.exists` - deprecated since v1
```

Comment:
Since `fs.exists` is deprecated, I think we should suggest either using the synchronous version (`fs.existsSync`) or inlining what `fs-extra` does (and Node docs recommend): `fs.access().then(() => true).catch(() => false)`.